app/task_routes.py

```diff
@@ -0,0 +1,97 @@
+from flask import Blueprint
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, make_response, request, abort
+from sqlalchemy import text
+from datetime import datetime
+import os, requests
+from .routes_helper import validate_model
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix = "/tasks")
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    if "title" not in request_body or "description" not in request_body:
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+
+    new_task = Task.from_dict(request_body)
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    response_body = dict(task = new_task.to_dict())
+
+    return jsonify(response_body), 201
+
+@tasks_bp.route("", methods=["GET"])
+def get_all_tasks():
+
+    if request.args.get("sort") == "asc":
+        tasks = Task.query.order_by(text("title asc"))
```

Comment:
We actually don't need to import `text` from sqlalchemy to get sorting by title. We can write: 
```python
task_query = Task.query.order_by(Task.title.asc())
# or
task_query = Task.query.order_by(Task.title.desc())